client/clients/tso/dispatcher.go

```diff
@@ -636,3 +636,10 @@ func (td *tsoDispatcher) checkTSORPCConcurrency(ctx context.Context, maxBatchWai
 func (td *tsoDispatcher) isConcurrentRPCEnabled() bool {
 	return td.rpcConcurrency > 1
 }
+
+// closeContext closes the connection context manager. This is used for testing purpose.
+func (td *tsoDispatcher) closeContext() {
+	cctx := td.provider.getConnectionCtxMgr()
+	cc := cctx.RandomlyPick()
```

Comment:
Why using `RandomlyPick`?

Reply:
the test case only  has one stream

Reply:
I prefer using `(c *Manager[T]) GetConnectionCtx(url string) *ConnectionCtx[T]` here.

Reply:
done